media/src/M.svelte

```diff
@@ -1,14 +1,30 @@
 <script>
-    import katex from "katex";
     import { afterUpdate, onMount } from "svelte";
 
     let span, span2;
+    export let display = false;
+    export let size = '';
 
-    const render = () =>
-          katex.render(span.innerText, span2, {
-              throwOnError: false,
-              displayMode: false,
-          });
+    const formatMJ = function(formula, display, size) {
+        let text = '\\Large';
+        if (size === 'sm') {
+            text = '';
+        }
+        if (display) {
+            return `$$ \\Large ${formula} $$`;
+        } else {
+            return `\\( ${text} ${formula} \\)`;
+        }
+    }
+
+    const render = () => {
+        span2.innerHTML = formatMJ(
+            span.innerText,
+            display,
+            size
+            );
+        MathJax.typesetPromise();   // eslint-disable-line
```

Comment:
Instead of disabling eslint for this line, it's better to declare the MathJax global variable for eslint at the top of the file, e.g.:
```
/* global MathJax */
```

https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-globals